templates/partials/answers/duration.html

```diff
@@ -37,7 +33,19 @@
             "text": format_unit_input_label('duration-month', unit_length='long') | capitalize
           }
         }) }}
-      </div>
     {% endif %}
   </div>
-{% endcall %}
+{%- endset -%}
+
+{%- if mutuallyExclusive -%}
+  {% set mutuallyExclusive = mutuallyExclusive | setAttributes({
+    "legend": question_title,
+    "legendClasses": "u-vh"
+  }) -%}
+
+  {%- call onsMutuallyExclusive(mutuallyExclusive) -%}
+    {{- fields | safe -}}
+  {%- endcall -%}
+{%- else -%}
+  {{- fields | safe -}}
+{%- endif -%}
```

Comment:
Possibly not as part of this pr, but can we make duration in input type like `onsDateInput`? This would mean the mutually exclusive could be dealt with in the same way as other inputs.

Reply:
Yeah probably out of scope for this PR as will require a new design system release and we're already 1 major version ahead of the version we're updating to in this PR, but it's definitely something we can do!